sample-app/src/components/DropdownSection.tsx

```diff
@@ -1,18 +1,18 @@
 import classNames from "classnames";
-import { useState, useEffect } from "react";
+import { useState, useEffect, useRef } from "react";
 
 export interface Option {
   value: string,
   display: JSX.Element
 }
 
 export interface DropdownSectionProps {
-  focusStatus?: string,
+  isFocused?: boolean,
   options: Option[],
   optionIdPrefix: string,
   onFocusChange?: (value: string, focusedOptionId: string) => void,
   onLeaveSectionFocus?: (pastSectionEnd: boolean) => void,
-  onClickOption?: (option: Option, optionIndex: number) => void,
+  onSelect?: (optionValue: string, optionIndex: number) => void,
```

Comment:
Minor nit: could we call this `onSelectOption`? I want to differentiate between selecting a section and an option.